tests/bench/bench.ml

```diff
@@ -20,37 +20,45 @@ let () =
   in
   let time_misspelled_cmd =
     (* NOTE: https://github.com/ocaml/opam/issues/5479 *)
+    Gc.compact ();
     time_cmd ~exit:2 (fmt "%s sitch" bin)
   in
   let time_install_cmd =
     (* NOTE: https://github.com/ocaml/opam/issues/5502 *)
+    Gc.compact ();
     launch "opam switch create one --empty";
     time_cmd ~exit:0 (fmt "%s install magic-trace -y --fake --sw one" bin)
   in
   let time_install_cmd_w_invariant =
     (* NOTE: https://github.com/ocaml/opam/issues/5502 *)
+    Gc.compact ();
     launch "opam switch create two --empty";
     launch "opam switch set-invariant core -n --sw two";
     time_cmd ~exit:0 (fmt "%s install magic-trace -y --fake --sw two" bin)
   in
-  let time_OpamSystem_read_10 =
+  let time_OpamSystem_read_100 =
```

Comment:
This reading bench test concerns reading a lot of opam files. Do we want to have also a test for reading huge file too? Currently (with this hash), `all-opam-files` file is 1.3M.

Reply:
i don't think so given the overwhelming present use-case of `OpamSystem.read`. Also i don't think this should be done in this PR if we'd want to do that. This PR is only for stability and performance improvements.

Reply:
fair enough